### DISCUSSION TITLE
some unreasonable results when reproduce

### DISCUSSION THREAD

**Discussion by LeighSu** [yoshitomo-matsubara/torchdistill#372] [Q&A]
I am trying to add new self-defined loss function, but when I want to compare the result with the existed loss function, the reproduced result seems unreasonable.
I used the following two command to reproduce  the result, and the teacher model seems normally, it can archive high accurate.
![image](https://github.com/yoshitomo-matsubara/torchdistill/assets/13862696/d14aa612-3154-427e-a9a6-a597e938f1f1)
but the student model only 8% acc for Top1 and 22% acc for Top 5.
![image](https://github.com/yoshitomo-matsubara/torchdistill/assets/13862696/e108127a-353c-4248-8a06-7517bf77b23e)
 **Is there anyting I misunderstand or how can I improve?**
 
[ilsvrc2012_cckd_resnet18_from_resnet50.txt](https://github.com/yoshitomo-matsubara/torchdistill/files/11991022/ilsvrc2012_cckd_resnet18_from_resnet50.txt)
[ilsvrc2012_kd_resnet18_from_resnet152.txt](https://github.com/yoshitomo-matsubara/torchdistill/files/11991023/ilsvrc2012_kd_resnet18_from_resnet152.txt)

python3 -m torch.distributed.launch --nproc_per_node=2 --use_env examples/image_classification.py --world_size 2 --config configs/sample/ilsvrc2012/single_stage/kd/resnet18_from_resnet152.yaml --log log/ilsvrc2012/kd/resnet18_from_resnet152.txt

python3 -m torch.distributed.launch --nproc_per_node=2 --use_env examples/image_classification.py --world_size 2 --config configs/sample/ilsvrc2012/single_stage/cckd/resnet18_from_resnet50.yaml --log log/ilsvrc2012/cckd/resnet18_from_resnet50.txt

Addition message:
I checkout the branch t0 tag v0.3.3 .
![image](https://github.com/yoshitomo-matsubara/torchdistill/assets/13862696/1e8ac8f8-323d-44a6-b109-c3890e61e188)

**Comment by yoshitomo-matsubara** [comment]
Hi @LeighSu 

What results are you trying to reproduce?

I believe you're using **sample configs for ImageNet datasets**, which are not tuned at all or whose results are not reported in some papers either.

Please read [configs/legacy/README.md](https://github.com/yoshitomo-matsubara/torchdistill/tree/main/configs/legacy) 

> Most of the configurations in `sample/` should be used for debugging or as starter configs to design your experiments.
If you use the methods as baselines, you **SHOULD** tune the hyperparameters in the configuration for a fair comparison.
